pages/administratorQuery/administratorQuery.js

```diff
@@ -4,6 +4,7 @@ const asyncHandler = require('express-async-handler');
 const fsPromises = require('fs').promises;
 const path = require('path');
 const _ = require('lodash');
+const hljs = require('highlight.js');
```

Comment:
```suggestion
const hljs = require('highlight.js/lib/highlight.js');
hljs.registerLanguage('sql', require('highlight.js/lib/languages/sql'));
```

I'm thinking we should pull in only the sql language here

Reply:
I don't mind doing this, but is there really an advantage? We load `highlight` only once when node starts up, and because other pages (`instructorFileBrowser`) are already loading it we get it for free (Node's `require()` only imports one shared copy of each package for the entire process).

Reply:
Ah, didn't realize we we're already pulling everything in &mdash; in that case I say go ahead and import it all :slightly_smiling_face: